src/modules/common/components/bars/topBar/hardwareWallet.css

```diff
@@ -0,0 +1,156 @@
+@import '../../../../../../setup/react/app/mixins.css';
+
+.wrapper {
+  margin-right: 16px;
+  position: relative;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+
+  & .hwIcon {
+    width: 20px;
```

Comment:
also
`cursor: pointer;`

Reply:
reason I would not want to add this is because it could miss lead users thinking it has a clickable action mean while it doesn't

Reply:
But it has a clickable action. Wont clicking the icon open upp the dropdown?